avocado/core/loader.py

```diff
@@ -411,15 +412,17 @@ def add_loader_options(parser, section='run'):
                                     parser=arggrp,
                                     long_arg='--loaders')
 
-    arggrp.add_argument('--external-runner', default=None,
-                        metavar='EXECUTABLE',
-                        help=('Path to an specific test runner that '
-                              'allows the use of its own tests. This '
-                              'should be used for running tests that '
-                              'do not conform to Avocado\' SIMPLE test'
-                              'interface and can not run standalone. Note: '
-                              'the use of --external-runner overwrites the --'
-                              'loaders to "external_runner"'))
+    help_msg = ("Path to an specific test runner that allows the use of its "
+                "own tests. This should be used for running tests that do not "
+                "conform to Avocado\' SIMPLE test interface and can not run "
```

Comment:
It was not introduced by your commit, but this should read: "conform to Avocado's SIMPLE test..."